.vscode/tasks.json

```diff
@@ -0,0 +1,67 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build PCSX2 - Debug GDB",
+            "type": "shell",
+            "command": "cd build && cmake -DEGL_API=TRUE -DCMAKE_BUILD_TYPE=Debug -GNinja -DBUILD_REPLAY_LOADERS=TRUE -DCMAKE_BUILD_PO=FALSE .. && ninja -j 10 && ninja install && cd ..",
```

Comment:
Hardcoding the job count doesn't seem like a great idea to me. I have more than 10 CPUs ;)

Is there any way to only run the cmake command once? After it's been generated for the first time, there's no need to run it every time; it'll automatically regenerate the build file when needed by just invoking ninja. Running it unconditionally just adds a few seconds to the launch time.

Reply:
Ninja uses all CPU threads by default, so you do not need to pass `-j` to the build command.

Also, I recommend using the build system-agnostic `cmake --build .` and `cmake --build . target install` instead of `ninja` and `ninja install`